mkosi

```diff
@@ -918,6 +918,10 @@ def run_workspace_command(args, workspace, *cmd, network=False, env={}, nspawn_p
     else:
         cmdline += ["--private-network"]
 
+    # Fix systemd/mkosi#245
```

Comment:
And no comments like this... If you are going to add a comment, tell why you need to modify the environment, not a link to an issue